src/controller/user.py

```
@@ -58,7 +58,7 @@ async def create_user_ctrl(session: SessionDep, body: BodyCreateUser) -> User:
 async def enroll_user_ctrl(session: SessionDep, user_id: str) -> StandbyReqTbl:
     user = session.get(User, user_id)
     if not user: raise HTTPException(404, detail="user not found")
-    if user.status != UserStatus.pending: raise HTTPException(400, detail="Only user with pending status can enroll")
+    if user.status != UserStatus.pending:raise HTTPException(400, detail="Only user with pending status can enroll")
```

Ruff (0.13.3)

Multiple statements on one line (colon)

(E701)